src/30_Advanced/AMP_cache.html

```diff
@@ -0,0 +1,56 @@
+<!--
+  #### Introduction
+
+  Google provides a cache that clients can use for free to display AMP files. This gives you two choices to display an AMP file in your app using either:
+
+  - the version hosted by the publisher (ampUrl) or
+
+  - the version hosted in Google’s AMP Cache (cdnAmpUrl)
+
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### How to access the AMP cache with HTTP url -->
+  <!--
+    Prepend `cdn.ampproject.org/c/` to AMP Urls. 
+  -->
+    <a href="http://www.bbc.co.uk/news/amp/35838735">Link with AMP HTTP URL</a>
+    <a href="https://cdn.ampproject.org/c/www.bbc.co.uk/news/amp/35838735">Link with AMP HTTP URL accessing the cache</a>
+
+  <!-- #### How to access the AMP cache with HTTPS url -->
+  <!--
+    Prepend `cdn.ampproject.org/c/s` to AMP Urls. 
+
+  -->
+    <a href="https://ampbyexample.com/components/amp-video/">Link with AMP HTTPS URL</a>
+    <a href="https://cdn.ampproject.org/c/s/ampbyexample.com/components/amp-video/">Link with AMP HTTPS URL accessing the cache</a>
+    
+    <!-- #### GET Arguments -->
+    <!--
+      GET arguments are supported when using AMP cache.
+    -->
+
+    <a href="https://cdn.ampproject.org/c/www.standard.co.uk/showbiz/celebrity-news/harry-styles-officially-added-to-cast-list-for-christopher-nolan-s-dunkirk-a3204886.html?amp">Link containing a GET argument.</a>
+    
+    <!-- #### Redirect -->
+    <!--
+      Redirect is supported when using AMP cache.
+    -->
+    <a href="https://cdn.ampproject.org/c/s/ampbyexample.com/amp-img.html">Link which will result in a redirect to https://ampbyexample.com/components/amp-img/.</a>
+    
+    <!-- #### Not Found Page -->
+    <!--
+      When a page is not found in the amp cache, 404 will be returned.
```

Comment:
When a page is not found in the amp cache, the AMP Cache will show an error page and return a 404 error.